gui/modlistman.lua

```diff
@@ -0,0 +1,446 @@
+                            function()
+                                remove_preset(existing_idx)
+                                rename_preset(idx, t)
+                                refresh_list()
+                            end
+                        )
+                    else
+                        rename_preset(idx, t)
+                        refresh_list()
+                    end
+                end)
+            end,
+        },
+
+        widgets.HotkeyLabel{
+            frame = { l = 1, b = 2, w = 15 },
+            key = "SELECT",
+            label = "Load",
+            on_activate = function()
+                local idx, current = presetList:getSelected()
+
+                if not idx then
+                    return
+                end
+
+                load_preset(idx)
+            end,
+        },
+
+        widgets.HotkeyLabel{
+            frame = { l = 17, b = 2, w = 15 },
+            key = "CUSTOM_D",
+            label = "Delete",
+            on_activate = function()
+                local idx, current = presetList:getSelected()
+
+                if not idx then
+                    return
+                end
+
+                dialogs.showYesNoPrompt(
+                    "Confirmation",
+                    "Delete " .. current.text .. "?",
+                    nil,
+                    function()
+                        remove_preset(idx)
+                        refresh_list()
+                    end
+                )
+            end,
+        },
+
+        widgets.HotkeyLabel{
+            frame = { l = 1, b = 0, w = 20 },
+            key = "CUSTOM_Q",
+            label = "Set as default",
+            on_activate = function()
+                local idx, current = presetList:getSelected()
+
+                if not idx then
+                    return
+                end
+
+                toggle_default(idx)
+                refresh_list()
+            end
+        },
+    }
+end
+
+ModmanageScreen = defclass(ModmanageScreen, gui.ZScreen)
+ModmanageScreen.ATTRS {
+    focus_path = "modman_screen",
+    defocusable = false,
+}
+
+function ModmanageScreen:init()
+    self:addviews{
+        ModmanageMenu{}
+    }
+end
+
+ModmanageOverlay = defclass(ModmanageOverlay, overlay.OverlayWidget)
+ModmanageOverlay.ATTRS {
+    frame = { w=30, h=1 },
+    default_pos = {x=5, y=-5},
+    viewscreens = {"new_region"},
+    default_enabled=true,
+    overlay_only=true,
+    hotspot=true,
+    overlay_onupdate_max_freq_seconds=0,
+}
+
+function ModmanageOverlay:init()
+    self:addviews{
+        widgets.HotkeyLabel{
+            frame = {l=0, t=0, w = 20, h = 1},
+            key = "CUSTOM_M",
+            view_id = "modman_open",
+            label = "Mod Manager",
+            on_activate = function()
+                ModmanageScreen{}:show()
+            end,
+        },
+    }
+end
+
+function ModmanageOverlay:overlay_onupdate(viewscreen)
+    self.visible = is_modlist_visible()
```

Comment:
Let me add a focus string that covers this case so you don't have to manage this manually.